src/harness/sas.cfg

```diff
@@ -1,4 +1,7 @@
 [SasConfig]
-BaseUrl: localhost:9000
-Version: v1.2
-
+AdminAndCbsdSasRsaBaseUrl: localhost:9000
```

Comment:
Can you split into `AdminBaseUrl` and `CbsdSasRsaBaseUrl`? It's confusing to have the two mixed together.

Reply:
yes sure, I updated the code.